inventory/byo/hosts.ose.example

```diff
@@ -91,6 +91,22 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # LDAP auth
 #openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'preferredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': '', 'insecure': 'false', 'url': 'ldap://ldap.example.com:389/ou=users,dc=example,dc=com?uid'}]
 
+# Cloud provider facts
+#
```

Comment:
We're already pulling the zone info out of metadata so I'm just reusing
here.

On Monday, March 21, 2016, Stefanie Forrester <notifications@github.com>
wrote:

> In inventory/byo/hosts.ose.example
> <https://github.com/openshift/openshift-ansible/pull/1634#discussion_r56908288>
> :
>
> > @@ -91,6 +91,22 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
> >  # LDAP auth
> >  #openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'preferredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': '', 'insecure': 'false', 'url': 'ldap://ldap.example.com:389/ou=users,dc=example,dc=com?uid'}]
> >
> > +# Cloud provider facts
> > +#
>
> This looks great! Would I also put my zone in here? Just trying to get a
> head start on the next build... :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1634/files/82c41633c931e6a3821867b5c5adb385b260c892#r56908288>
>